tests/unit_tests/quantization_utility_tests.py

```diff
@@ -0,0 +1,358 @@
+import unittest
+import torch
+from torch import nn
+
+try:
+    from pytorch_quantization import nn as quant_nn
+    from super_gradients.training.utils.quantization.fine_grain_quantization_utils import QuantizationUtility, \
+        RegisterQuantizedModule
+    from super_gradients.training.utils.quantization.core import SkipQuantization, SGQuantMixin, QuantizedMapping
+
+    _imported_pytorch_quantization_failure = None
+
+except (ImportError, NameError, ModuleNotFoundError) as import_err:
+    _imported_pytorch_quantization_failure = import_err
+
+
+@unittest.skipIf(_imported_pytorch_quantization_failure is not None, "Failed to import `pytorch_quantization`")
+class QuantizationUtilityTest(unittest.TestCase):
```

Comment:
We need to add some example code for usage. Can't assume users will search the tests directory for usage example.

Reply:
Added examples in [3c93d62](https://github.com/Deci-AI/super-gradients/pull/304/commits/3c93d622af99ac6d68ba532b5c8f939ef67ef20e)